addOns/scripts/src/main/javahelp/org/zaproxy/zap/extension/scripts/resources/help/contents/options.html

```diff
@@ -19,6 +19,16 @@ <h2>When the Script in the Console Changes on Disk</h2>
 
 Note that if there are unsaved changes to the script, you will always be prompted to choose which version to keep.
 
+<h2>Font</h2>
+
+<h3>Font Name</h3>
+<p>Set the name of the font used in the script console.
+    The default font is <code>Monospaced</code>.
```

Comment:
Is this a default (base install) font across Windows and MacOS as well?

Reply:
Yup. From https://docs.oracle.com/javase/tutorial/2d/text/fonts.html#logical-fonts,
> These fonts are available on any Java platform and can be thought of as aliases for some underlying font that has the properties implied by its name.

Reply:
Oh okay. I had looked at some other docs for Windows but if it's an alias name then that makes more sense.

Reply:
Tweaked this for clarity.